README.md

```diff
@@ -1,24 +1,187 @@
+- **[Testing]** All the views are being tested using [RSpec](https://rspec.info/) and [Capybara](https://github.com/teamcapybara/capybara)
+- **[Linters]** It follows the rules of 2 very well knowed linters [Rubocop](https://rubocop.org/) & [Stylelint](https://stylelint.io/), which helps to keep the code clean and easy to read
+- **[Gitflow]** Used [Gitflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) as a workflow which allows to have a clean and organized version control
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+<!-- ## 🚀 Preview <a name="live-demo"></a>
+
+Will be implemented in the future -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ To have ruby installed with the last version of rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  Run bundle install to install all the gems and dependencies
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  Executing the command 'rails s' will start the server of this RoR app
+```
+
+### Testing
+
+To run integration tests, you have to follow these steps:
+
+```sh
+  Executing the command `bundle install` will install all the necessary dependencies to run the tests of this RoR app
+```
+
+```sh
+  To run all the tests, execute the command 'rspec'
+```
+<!-- 
+```sh
+  To run specific the specific tests for each view, execute one of the following codes:
+
+  Do user_index view test command: 'rspec spec/views/users/index.html.erb_spec.rb'
+  Do user_show view test command: 'rspec spec/views/users/show.html.erb_spec.rb'
+  Do posts_index view test command: 'rspec spec/views/posts/index.html.erb_spec.rb'
+  Do posts_show view test command: 'rspec spec/views/posts/show.html.erb_spec.rb'
+``` -->
+
+### Deployment
+
+No deployment necessary
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+## 👥 Author <a name="author"></a>
+
+👤 **Álvaro Iporre Martínez**
+
+- GitHub: [@alvaroiporre](https://github.com/alvaroiporre)
+- Twitter: [@alvaro.iporre](https://twitter.com/Alvaro41065582)
+- LinkedIn: [Álvaro Iporre](https://www.linkedin.com/in/alvaroiporre/)
+
+<!-- ## ✨ Contributors
+ -->
+
+
+<!-- FEATURES -->
+
+## 🔭 Future Features <a name="features"></a>
+
+- **[Responsive Design]** Build a fully responsive design
+- **[Live Demo]** Deploy the app in a live server
+- **[User Profile]** Add a user profile page
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```

Comment:
![image](https://github.com/alvaroiporre/Tech-Talent-Potosi/assets/93039359/12e3f146-ae7c-4b79-acf1-f459c25f0ed3)

- [ ] It would be nice if you fix your Issues page link to make your readme descriptive and professional because right now I am getting the above error when I click this link 😨